lms/views/exceptions.py

```diff
@@ -27,6 +27,13 @@ def notfound(self):
 
     @forbidden_view_config()
     def forbidden(self):
+        if validation_error := getattr(self.exception.result, "validation_error", None):
```

Comment:
Does `exception.result` always exist or might this crash in some circumstances?

Reply:
Seems to be part of the exception object in all cases: 

https://docs.pylonsproject.org/projects/pyramid/en/latest/api/httpexceptions.html#pyramid.httpexceptions.HTTPForbidden